lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test', user: 'postgres', password: 'postgres')
```

Comment:
the inclusion of **user**: and **password**: meant that I couldn't actually rackup. I had to replace the connect statement with this, for it to work.
  
**connection = PG.connect(dbname: 'chitter_test')**